lib/tasks/support.rake

```diff
@@ -131,9 +131,9 @@ namespace :support do
   end
 
   desc "Send a test email to an email address"
-  task :deliver_to_test_email, [:test_email_address] => :environment do |_t, args|
+  task :send_test_email, [:email_address] => :environment do |_t, args|
```

Comment:
This will need an update [here](https://github.com/alphagov/email-alert-api/blob/master/docs/support-tasks.mdl).

Reply:
That is already part of https://github.com/alphagov/email-alert-api/pull/1438/commits/3075e03d8ae4f3f612112ec687ca36d03acc9477

Reply:
Oh sorry - I confused myself going in a loop through the dev docs!